lib/smack/Prelude.cpp

```diff
@@ -338,25 +399,29 @@ std::string Prelude::getPrelude() {
     s << "const $" << i << ".ref: ref;" << "\n";
     t << "$" << i << ".ref";
     s << Decl::axiom(Expr::eq(Expr::id(t.str()),
-      rep.pointerLit((unsigned long long) i)))
+          rep.pointerLit((unsigned long long) i)))
```

Comment:
I don't understand this reformatting here and why it is better. Is this how code would get formatted in the LLVM source?
More specifically, is this what `clang-format` would do?
To me the original formatting here make more sense, but maybe you looked at the LLVM style guidelines and LLVM source code and learned that your changes here adhere to the LLVM formatting standard.

We should be running `clang-format` on our CI: https://github.com/smackers/smack/issues/429

Reply:
I used auto indentation provided by vim. I should run clang-format.